addons/l10n_es_edi_tbai/models/account_move.py

```diff
@@ -0,0 +1,254 @@
+
+    def _get_l10n_es_tbai_values_from_xml(self, xpaths):
+        """
+        This function reads values directly from the 'post' XML submitted to the government \
+            (the 'cancel' XML is ignored).
+        """
+        res = {key: '' for key in xpaths.keys()}
+        doc_xml = self._get_l10n_es_tbai_submitted_xml()
+        if doc_xml is None:
+            return res
+        for key, value in xpaths.items():
+            res[key] = doc_xml.find(value).text
+        return res
+
+    @api.depends('company_id', 'edi_state', 'l10n_es_tbai_post_xml_id')
+    def _compute_l10n_es_tbai_values(self):
+        """
+        This function reads values directly from the 'post' XMLs submitted to the government \
+            (the 'cancel' XML is ignored).
+        Note about the signature value: if the post XML has \
+            not been validated (yet), the signature value is left blank.
+        """
+        for record in self:
+            vals = record._get_l10n_es_tbai_values_from_xml({
+                'signature': r'.//{http://www.w3.org/2000/09/xmldsig#}SignatureValue',
+                'registration_date': r'.//CabeceraFactura//FechaExpedicionFactura'
+            })
+
+            if record.edi_state not in ('sent', 'cancelled'):
+                # Documents may exist but have not been validated by govt (eg. timeout)
+                record.l10n_es_tbai_signature = ''
+            else:
+                # RFC2045 - Base64 Content-Transfer-Encoding (page 25)
+                # Any characters outside of the base64 alphabet are to be ignored in base64-encoded data.
+                record.l10n_es_tbai_signature = vals['signature'].replace("\n", "")
+            if vals['registration_date']:
+                record.l10n_es_tbai_registration_date = datetime.strptime(vals['registration_date'], '%d-%m-%Y').replace(tzinfo=timezone('Europe/Madrid'))
+            else:
+                record.l10n_es_tbai_registration_date = False
+
+    @api.depends('name')
+    def _compute_l10n_es_tbai_sequence(self):
+        for record in self:
+            sequence, _ = record.name.rsplit('/', 1)
+            sequence = regex_sub(r"[^0-9A-Za-z.\_\-\/]", "", sequence)  # remove forbidden characters
+            sequence = regex_sub(r"[\s]+", " ", sequence)  # no more than once consecutive whitespace allowed
+            # TODO (optional) issue warning if sequence uses chars out of ([0123456789ABCDEFGHJKLMNPQRSTUVXYZ.\_\-\/ ])
+            self.l10n_es_tbai_sequence = sequence + ("TEST" if record.company_id.l10n_es_edi_test_env else "")
+
+    @api.depends('name')
+    def _compute_l10n_es_tbai_number(self):
+        for record in self:
+            _, number = record.name.rsplit('/', 1)
+            self.l10n_es_tbai_number = regex_sub(r"[^0-9]", "", number)  # remove non-decimal characters
+
+    def _is_l10n_es_tbai_simplified(self):
+        return self.commercial_partner_id == self.env.ref("l10n_es_edi_sii.partner_simplified")
```

Comment:
SII doesn't have much in account_move and only makes this check once, not sure where's the best place to put it. 
Anyway, this definition of "simplified partner" is not compatible with TicketBAI that allows simplified partners with actual names and directions. So it will have to change either way, not sure if SII will.